src/tutorials/race_timer.md

```diff
@@ -0,0 +1,4383 @@
+
+    The split needs 3 **UI Text** objects. Make sure to name them exactly as below, because the script will be looking for those 3 objects based on their name.
+
+    1. Split Name
+    2. Split Time
+    3. Race Time
+
+   ![!Split UI Hierarchy](../img/RaceTimerTutorial/split_ui.png){: .center loading="lazy" }
+
+   ![!Split UI Design](../img/RaceTimerTutorial/split_ui_design.png){: .center loading="lazy" }
+
+3. Right click on the **Split Entry** object and select **Create New Template From This**.
+
+    ![!Split Entry](../img/RaceTimerTutorial/split_entry.png){: .center loading="lazy" }
+
+4. Delete the **Split Entry** template from the **Hierarchy** so the **Splits** panel is now empty.
+
+We now have a template that will be used later for adding to the UI dynamically for each split. This saves time manually adding all the splits to the UI and positioning them.
+
+### Create Time Split Triggers
+
+1. In the **Triggers** group, create a new group called **Splits**.
+2. Add new triggers to the **Splits** group and place them on the track.
+
+!!! tip
+    Create as many triggers as you need. These triggers can be named to something more interesting as they will be displayed in the UI. Make sure that players can't get around the triggers by making them oversized to cover all the objects (track and rails in this case).
+
+<div class="mt-video" style="width:100%">
+    <video autoplay muted playsinline controls loop class="center" style="width:100%">
+        <source src="/img/RaceTimerTutorial/split_triggers.mp4" type="video/mp4" />
+    </video>
+</div>
+
+### Update Race Manager Server Script
+
+We need to modify the **Race_Manager_Server** script so that it keeps track of what split the place has gone through. The script needs to know about the split triggers.
+
+1. Click the **Race_Manager_Server** script to make it the active object in the **Hierarchy**.
+2. Drag and drop the **Splits** group onto the **Add Custom Property** button.
+3. Rename the custom property to **splitTriggers**.
+
+```lua linenums="1"
+local SPLIT_TRIGGERS = script:GetCustomProperty("splitTriggers"):WaitForObject()
+```
+
+Add the above code just below `FINISH_TRIGGER`. This will allow easy access to all the split triggers.
+
+```lua linenums="1" hl_lines="10"
+local function OnPlayerJoined(player)
+    player.maxJumpCount = 0
+    player.canMount = false
+
+    players[player.id] = {
+
+        inRace = false,
+        startTime = 0,
+        finishTime = 0,
+        splits = {}
+
+    }
+
+    local data = Storage.GetPlayerData(player) or {}
+
+    if data.bestTime ~= nil then
+        SendPrivateData(player, "bestTime", data.bestTime)
+    end
+end
+```
+
+Add the code on line 10. We need to keep track of the splits the player has overlapped. To do this we can store them in a table called `splits` for each player.
```

Comment:
I was a little bit on the fence with specifying the script, as the section header states which script it is they are modifying.  However, I can see how easily it could be for the user to forget which script it is they are modifying and needing to scroll back up to see the header.